sites/lab.zooniverse.org.conf

```diff
@@ -2,20 +2,24 @@ server {
     include /etc/nginx/ssl.default.conf;
     server_name lab.zooniverse.org;
 
-    location ~ \.(js|css)$ {
+    location / {
         resolver 8.8.8.8;
-        proxy_pass             http://zooniverse-static.s3-website-us-east-1.amazonaws.com/lab.zooniverse.org$request_uri;
-        include /etc/nginx/proxy-headers.conf;
-    }
+        proxy_set_header       Host zooniversestatic.z13.web.core.windows.net;
+        proxy_http_version 1.1;
 
-    location / {
-        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/lab.zooniverse.org$request_uri;
+        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip|js|css)$ https://zooniversestatic.z13.web.core.windows.net/lab.zooniverse.org$request_uri;
 
-        resolver 8.8.8.8;
+        add_header 'X-Content-Type-Options' 'nosniff';
+        add_header 'Content-Security-Policy' "frame-ancestors 'self'";
+        add_header 'X-Content-Security-Policy' "frame-ancestors 'self'";
+        add_header 'X-XSS-Protection' '1; mode=block';
+        proxy_cache            STATIC;
+        proxy_cache_valid      200  1d;
+        proxy_cache_use_stale  error timeout invalid_header updating
+                    http_500 http_502 http_503 http_504;
```

Comment:
not blocking - OOI is the long term strategy to inline the `proxy-headers.conf` directives everywhere? 

Will this harm re-use? 

Is it easier to manage changes to the single source file as it's setup now?

Reply:
I planned to replace existing proxy-headers.conf contents with these headers and re-add the include. But I do keep adding these headers in-line for each one, and it does make more sense to just create a new one and migrate to it as I go. I'll create a new PR before the next one of these to do that. 👍